.teamcity/src/subprojects/build/samples/ProjectSamples.kt

```diff
@@ -9,7 +9,19 @@ import subprojects.build.core.*
 import subprojects.release.*
 import subprojects.release.publishing.*
 
-val gradleProjects = listOf("client-mpp", "fullstack-mpp", "generic")
+data class SampleProjectSettings(val projectName: String, val vcsRoot: VcsRoot)
+
+val gradleProjects = listOf(
```

Comment:
I think you could drop the "ktor" prefix from every entry as this is the list of entries that will appear in the TeamCity tree branch...

Reply:
done